### PR TITLE
Alligning code to changes in CEF eIDAS Node 2.5

### LINKED ISSUE #103: Alligning with CEF eIDAS Node 2.5
New version od CEF eIDAS Node 2.5 has been released https://ec.europa.eu/cefdigital/wiki/display/CEFDIGITAL/eIDAS-Node+version+2.5 and there is some work that needs to be done to align with this. Here is the list of changes after analysing migration guide.
- There is new spCountryCode field in LightRequest/LightResponse that includes what was previously temporarily included in CitizenCountryCode as the two meaning field. This has been already implemented but under different name originCountryCode so the only change is to rename originCountryCode to spCountryCode.
- There is a new field RequesterId in LightRequest
- There is a new field Consent in LightResponse
- New code requires to use and also uses xml namespace lin LightRequest and LightResponse

### PR DESCRIPTION
**Pull request by jtalir**
Closes #103

### CODE REVIEW

File: eidas_node/models.py [hunk 1]
```diff
@@ -282,17 +288,17 @@ def deserialize_attributes(attributes_elm: Element) -> Dict[str, List[str]]:
     """Deserialize eIDAS attributes."""
     attributes = OrderedDict()  # type: Dict[str, List[str]]
     for attribute in attributes_elm:
-        if attribute.tag != 'attribute':
+        if attribute.tag.rpartition('}')[2] != 'attribute':
```
Comment (tpazderka) [inline eidas_node/models.py:291]: Rather than this, the RHS should be constructed with the namespace.

**Review by tpazderka — CHANGES_REQUESTED** [review]
Also the README needs to be changed to note that only version 2.5.x is supported.

File: eidas_node/models.py (hunk 1 above)
Comment (jtalir) [reply eidas_node/models.py:291]: This approach is compliant with both 2.4 and 2.5 version. It validates in both cases when XML is without namespace (2.4) and with namespace (2.5)

Comment (tpazderka) [reply eidas_node/models.py:291]: If I understand it correctly, then in 2.4 the namespace is empty and the result would be the same?

Comment (jtalir) [reply eidas_node/models.py]: Changed to QName(attribute.tag).localname

File: eidas_node/datamodels.py [hunk 2]
```diff
@@ -128,7 +131,8 @@ def export_xml(self) -> Element:
         self.validate()
         if not self.ROOT_ELEMENT:
             raise TypeError('XMLDataModel subclasses must define ROOT_ELEMENT class attribute.')
-        root = Element(self.ROOT_ELEMENT)
+        root_nsmap = {} if not self.ROOT_NS else {None: self.ROOT_NS}
+        root = Element(self.ROOT_ELEMENT, nsmap=root_nsmap)
```
Comment (fenryxo) [inline eidas_node/datamodels.py:135]: This combined with `ROOT_NS = 'http://cef.eidas.eu/LightResponse'` and `ROOT_NS = 'http://cef.eidas.eu/LightRequest'` is a breaking change, so we must specify that only eIDAS Node 2.5+ is supported in README.md.

**Review by fenryxo — CHANGES_REQUESTED** [review]
* [ ] README.md and CHANGELOG.md should be updated.
* [ ] Could you push your branch directly to the `CZ-NIC/django-eidas-specific-node` repository? It will be easier to run CI for it.
* [ ] We must fix CI before merging this PR.

File: eidas_node/datamodels.py (hunk 2 above)
Comment (jtalir) [reply eidas_node/datamodels.py:135]: eIDAS Node 2.4 apparently ignores namespace in LightRequest/LightResponse so this is not breaking change.

Comment (fenryxo) [reply eidas_node/datamodels.py:135]: Glad to hear that :-)

**Comment by jtalir** [comment]
Branch was pushed to main repo and new PR was created

**Pull request closed without merging**